macros/cross-adapter-modeling/base/segment_web_page_views.sql

```diff
@@ -7,7 +7,50 @@
 
 {% macro default__segment_web_page_views() %}
 
-{{ exceptions.raise_compiler_error("macro segment_web_page_views not implemented for this adapter") }}
+with source as (
+
+    select * from {{var('segment_page_views_table')}}
+    
+),
+
+renamed as (
+
+    select
+    
+        id as page_view_id,
+        anonymous_id,
+        user_id,
+        
+        received_at as received_at_tstamp,
+        sent_at as sent_at_tstamp,        
+        timestamp as tstamp,
+
+        url as page_url,
+        ltrim(split_part(split_part(url, '.com', 1), '//',2),'www.')::varchar 
+            as page_url_host,
+        path as page_url_path,
+        title as page_title,
+        search as page_url_query,
+        
+        referrer,
+        ltrim(split_part(split_part(referrer, '.com', 1), '//',2),'www.')::varchar 
+            as referrer_host,
+        
+        context_campaign_source as utm_source,
+        context_campaign_medium as utm_medium,
+        context_campaign_name as utm_campaign,
+        context_campaign_term as utm_term,
+        context_campaign_content as utm_content,
+        nullif(split_part(url, 'gclid=', 2)::varchar,'') as gclid,
```

Comment:
I don't think this works unless gclid is the final parameter. It will return everything after gclid, which is not a particularly safe approach. We can't rely on gclid always being the final section of a URL. Split on the `&` that follows.